engine/pom.xml

```diff
@@ -227,7 +227,7 @@
                             <archive>
                                 <manifestFile>${project.build.outputDirectory}/META-INF/MANIFEST.MF</manifestFile>
                                 <manifestEntries>
-                                    <Specification-Title>Jakarta Bean Validation</Specification-Title>
+                                    <Specification-Title>Jakarta Validation</Specification-Title>
                                     <Specification-Version>3.0</Specification-Version>
```

Comment:
Ah, so it's 3 after all?

Reply:
I've used the `build-helper-maven-plugin` to parse the jakarta validation version and use that where possible, so we wouldn't need to track where we must change the version 🙈